code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -113,6 +113,13 @@
 		reagents.reaction(user.loc)
 		reagents.clear_reagents()
 
+/obj/item/reagent_containers/spray/empty
+	lefthand_file = 'icons/mob/inhands/equipment/custodial_lefthand.dmi'
```

Comment:
Are these inhands not applied to the base spray bottle?

Reply:
They are not.
And, I just cpoied /obj/item/reagent_containers/spray/cleaner and removed reagents. That's it.

Reply:
Gotcha, checked out the types that this includes and it makes sense to me.